modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -459,3 +459,17 @@ bin/neo4j-admin database backup --to-path=azb://myStorageAccount/myContainer/myD
 ----
 ======
 =====
+
+
+[role=label--new-2025.04]
+[[diff-backup-as-parent]]
+=== Perform a differential backup using the `--prefer-diff-as-parent` option
+
+When creating a differential backup, the parent is always the most recent (non-empty) backup in the directory.
+In some cases the directory may have two equally recent backups -- a full backup and a differential one.
+By default, the full backup is chosen as the parent.
+But you can use the differential backup as the parent by setting the option `--prefer-diff-as-parent` to `True`.
```

Comment:
```suggestion
When creating a differential backup, the parent is usually the most recent (non-empty) backup in the directory.
In some cases you may prefer to use the latest differential backup as the parent, even if there is a more recent full backup.
You can use the latest differential backup as the parent by setting the option `--prefer-diff-as-parent` to `True`.
This can be used to ensure you have differential backups for all transactions, which would allow you to restore to any point in time.
Otherwise the transactions between a full backup and the previous differential backup will not be backed up as individual transactions.
```

Reply:
Thank you, @nick-giles-neo, for a super quick response 😄